domains/6666.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username":"uv301",
+    "email":"wupo4466@gmail.com"
+  },
+  "record":{
+    "AAAA":[
+      "2602:294:0:b7:1234:1234:0cfa:0001"
```

Comment:
Unable to view site, make sure you have an actual site and make sure all the boxes are completed in the pull request when you make it.